scanpy/external/pp/_magic.py

```diff
@@ -12,6 +12,9 @@
 from ..._compat import Literal
 
 
+__min_version__ = "2.0"
```

Comment:
Don’t use `__dunder__` naming for things that aren’t standardized python names. You can just use `UPPER_WITH_UNDERSCORES` for constants.